specification/monitor/Azure.Monitor.Query.Logs/main.tsp

```diff
@@ -0,0 +1,33 @@
+import "@typespec/http";
+import "./routes.tsp";
+
+using TypeSpec.Http;
+using TypeSpec.Versioning;
+
+@useAuth(
+  OAuth2Auth<[
+    {
+      type: OAuth2FlowType.implicit,
+      authorizationUrl: "https://login.microsoftonline.com/common/oauth2/authorize",
+      scopes: ["https://api.loganalytics.io/.default"],
+    }
+  ]>
+)
+@service(#{ title: "Monitor Query Logs" })
```

Comment:
Is changing the service name a breaking change? Know the service name [the swagger](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/operationalinsights/data-plane/Microsoft.OperationalInsights/stable/2022-10-27/OperationalInsights.json) has been using is `Operational Insights`.